debian/patches/0003-use-explicit-cxx-std-on-older-cmake.patch

```diff
@@ -0,0 +1,17 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -18,8 +18,12 @@
+ set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -DDEBUG")
+ 
+ # C++11 is required
+-set(CMAKE_CXX_STANDARD 11)
+-set(CXX_STANDARD_REQUIRED ON)
++if(CMAKE_COMPILER_IS_GNUCXX AND (CMAKE_VERSION VERSION_LESS 3.1))
```

Comment:
Feel free to change it as you please. I may not be able to spend much more time with this but will appreciate it if you can move it along. Alternatively if you like me to close this pull request, i am ok with that too.